helmfile/values/gatekeeper/podsecuritypolicies.gotmpl

```diff
@@ -0,0 +1,62 @@
+constraints:
+  ingress-nginx:
+    nginx:
+      labelsSelector:
+          app.kubernetes.io/component: controller
+      allow:
+        allowedCapabilities:
+          - NET_BIND_SERVICE
+        {{- if .Values.ingressNginx.controller.useHostPort }}
+        hostNetwork: true
+        {{- end }}
+        hostNamespace: true
+        privileged: true
```

Comment:
Are these really privileged? And this renders a constraint against privileged so I don't think it works as you intended. :no_mouth:

Reply:
You are right, I got confused when looking at ingress nginx :sweat_smile: I will fix that